iOS/Objective C/LevelPlayDemo.xcodeproj/project.pbxproj

```diff
@@ -8,13 +8,13 @@
 
 /* Begin PBXBuildFile section */
 		47B0AD2A1CB5252100B8D125 /* WebKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 47B0AD291CB5252100B8D125 /* WebKit.framework */; };
```

Comment:
you have leftovers in the project with bad references
![Screenshot 2025-06-22 at 14 43 17](https://github.com/user-attachments/assets/0f27226e-fafc-4b0e-b402-e8c3250119d5)

Reply:
it's after clean, Delete the "Objective C", reset it using git, run pod install and open the xc workspace. You should reproduce it then